Gemfile

```diff
@@ -23,7 +24,14 @@ group :development, :test do
   gem 'puppetlabs_spec_helper',  :require => false
   gem 'puppet-lint',             :require => false
   gem 'simplecov',               :require => false
-  gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  if beaker_version
+    gem 'beaker', *location_for(beaker_version)
+  end
+  if beaker_rspec_version
+    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  else
+    gem 'beaker-rspec',  :require => false
+  end
```

Comment:
Yes, @justinstoller is correct.  This doesn't alter the functionality/defaults of the Gemfile - just adds support for BEAKER_VERSION/BEAKER_RSPEC_VERSION.